src/Tests/Microsoft.NET.Build.Containers.IntegrationTests/EndToEndTests.cs

```diff
@@ -65,7 +67,7 @@ public async Task ApiEndToEndWithRegistryPushAndPull()
         var sourceReference = new ImageReference(registry, DockerRegistryManager.BaseImage, DockerRegistryManager.Net6ImageTag);
         var destinationReference = new ImageReference(registry, NewImageName(), "latest");
 
-        await registry.PushAsync(builtImage, sourceReference, destinationReference, Console.WriteLine, cancellationToken: default).ConfigureAwait(false);
+        await registry.PushAsync(builtImage, sourceReference, destinationReference, WriteMessage, cancellationToken: default).ConfigureAwait(false);
```

Comment:
minor: why is an actual cancellation token not passed?

Reply:
mostly because this is in testing code only, and pushing to the local dockerized registry is quite fast. cancellation is never a problem in tests, the main problem that arises is if the local docker registry started at all or not!